src/dags/3_batch_sqs/rabbitmq_producer.py

```diff
@@ -0,0 +1,111 @@
+"""Send generated data to RabbitMQ.
+
+Maintainer:
+    Name: Donghyun Kim
+    Email: rkdqus2006@naver.com
+"""
+from datetime import datetime, timedelta
+
+import pika
+from airflow.models import DAG
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import PythonOperator
+from pymongo import MongoClient
+
+
+def produce_data_to_queue(
+    host: str,
+    port: int,
+    user: str,
+    password: str,
+    queue_name: str = "rabbitmq-demo-queue",
```

Comment:
```suggestion
    queue_name: str = "rabbitmq-simple-queue",
```
demo -> simple 어떠세요?

References:
- AWS SQS (**Simple** Queue Service)
- **Simple** DataOps Docker

Reply:
변경했습니다 😎